06-documenting_R.Rmd

```diff
@@ -0,0 +1,110 @@
+    * Author(s) names should be separated by a comma or `and` (but not both).
+    * Separate paragraphs (separated by a blank line) should be used for each reference.
+    * Give a date immediatey after the author(s) names.
+    * Do not put a period after the date.
+    * Titles of books and journals (not articles) should be enclosed in `\emph{...}`.
+    * Volume numbers for journals are to be enclosed in `\bold{...}` and followed by a comma.
+    * Use `--` for page ranges.
+    * For given an address for a publisher use the format `New York: Springer-Verlag`.
+
+More guidelines for writing .Rd files can be found [here](https://developer.r-project.org/Rds.html).
+
+The language used in the documentations should follow these basic rules:
+
+1. Affirmative tone should be used to describe what the function does and how to use it effectively. Rather than creating worry in the mind of a reader, it should establish confident knowledge about the effective use of the particular function/feature.
+
+2. More documentation is not necessarily better documentation. Long descriptions full of corner cases and caveats can create the impression that a function is more complex or harder to use than it actually is. Be succinct but exhaustive. 
+
+3. Short code examples can help in understanding better. Readers can often grasp a simple example more quickly than they can digest a formal description. Usually people learn faster with concrete, motivating examples that match the context of a typical use case. 
+
+4. Giving a code equivalent (or approximate equivalent) can be a useful in addition to the description provided. You should carefully weigh whether the code equivalent adds value to the document.
+
+5. The tone of the documentation needs to be respectful of the reader’s background. Lay out the relevant information, show motivating use cases, provide glossary links, and do your best to connect-the-dots. The documentation is meant for newcomers, many of whom will be it to evaluate the R language as a whole. The experience needs to be positive and not leave the reader with worries that something bad will happen if they make a mistake. 
+
+Extensive details of writing R documentation files can be found [here](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Writing-R-documentation-files).
+
+## Translating
+
+### Starting a new translation
+
+## Helping with documentation
+
+Maintaining the accuracy  of R's documentations and keeping a high level of quality takes a lot of effort. Community members, like you, help with writing, editing, and updating content, and these contributions are appreciated and welcomed.
+
+Looking at pre-existing documentation source files can be very helpful when getting started.
+
+If you look at documentation issues on Bugzilla, you will find various documentation problems that may need work. Issues vary from typos to unclear documentation and items lacking documentation.

```

Comment:
Perhaps a link to the book's chapter how to review an issue on Bugzilla?

Reply:
It could be worth mentioning how to browse the documentation bugs on Bugzilla. Since "Documentation" is one of the possible components you can set in the metadata, you can browse by this component: https://bugs.r-project.org/bugzilla/buglist.cgi?component=Documentation